src/cc/bcc_bpf_uapi_inc.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#ifdef HAVE_EXTERNAL_LIBBPF
+# include <bpf/uapi/linux/bpf.h>
+# include <bpf/uapi/linux/bpf_common.h>
+# include <bpf/uapi/linux/btf.h>
```

Comment:
Is it possible that we have a cmake option like BCC_EXT_LIBBPF_INC_PREFIX for example
   BCC_EXT_LIBBPF_INC_PREFIX = "bpf/uapi"
so we can replace the above hardcoded prefix 'bpf/uapi' with the one passed from cmake system?